compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/ContextMenuProvider.desktop.kt

```diff
@@ -52,10 +52,39 @@ fun ContextMenuArea(
     state: ContextMenuState = remember { ContextMenuState() },
     enabled: Boolean = true,
     content: @Composable () -> Unit
+) {
+    ContextMenuArea(
+        items = items,
+        state = state,
+        modifier = Modifier.contextMenuOpenDetector(state, enabled),
+        content = content
+    )
+}
+
+/**
+ * Defines a container where context menu is available. Representation of menu is defined by
+ * [LocalContextMenuRepresentation].
+ *
+ * This overload does not trigger the opening of the context menu; it's up to the caller to do so
+ * by passing a [modifier] that does so.
+ *
+ * @param items List of context menu items. Final context menu contains all items from descendant
+ * [ContextMenuArea] and [ContextMenuDataProvider].
+ * @param state [ContextMenuState] of menu controlled by this area.
+ * @param modifier The modifier to attach to the element; this should include the trigger that opens
+ * the context menu (e.g. on right-click).
+ * @param content The content of the [ContextMenuArea].
+ */
+@Composable
+fun ContextMenuArea(
+    items: () -> List<ContextMenuItem>,
+    state: ContextMenuState,
+    modifier: Modifier,
```

Comment:
```suggestion
    state: ContextMenuState = remember { ContextMenuState() },
    modifier: Modifier = Modifier.contextMenuOpenDetector(state, true),
```

Reply:
I'm afraid that *this* would be confusing, because
1. The default value for a `modifier` argument is (nearly?) always `Modifier`.
2. `contextMenuOpenDetector` is not the modifier typically needed when using this overload. If you want `contextMenuOpenDetector`, you should use the overload right above it, that just takes an `enabled` flag.